tck/cmd/server.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/creachadair/jrpc2"
+	"github.com/creachadair/jrpc2/handler"
+	"github.com/creachadair/jrpc2/jhttp"
+	"github.com/hashgraph/hedera-sdk-go/tck/methods"
+	"github.com/hashgraph/hedera-sdk-go/tck/response"
+	"github.com/hashgraph/hedera-sdk-go/v2"
+	"github.com/joho/godotenv"
+)
+
+func main() {
+	// Load dotenv
+	_ = godotenv.Load()
+
+	// Initialize the SDK service
+	sdkService := new(methods.SDKService)
+	accountService := new(methods.AccountService)
+	accountService.SetSdkService(sdkService)
+
+	// Create a new RPC server
+	assigner := handler.Map{
+		"setup":         postHandler(HandleError, handler.New(sdkService.Setup)),
+		"reset":         postHandler(HandleError, handler.New(sdkService.Reset)),
+		"createAccount": postHandler(HandleError, handler.New(accountService.CreateAccount)),
+		"generateKey":   postHandler(HandleError, handler.New(methods.GenerateKey)),
+	}
+
+	bridge := jhttp.NewBridge(assigner, nil)
+
+	// Listen and redirect to bridge
+	http.HandleFunc("/", bridge.ServeHTTP)
+	port := os.Getenv("TCK_PORT")
+	if port == "" {
+		port = "80"
+	}
+	fmt.Println("Server is listening on port: " + port)
```

Comment:
Maybe at some point we can refactor to using log or logrus.

Reply:
changed